packages/react-composites/tests/browser/chat/live-tests/ChatComposite.test.ts

```diff
@@ -1,10 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { stubMessageTimestamps, waitForChatCompositeToLoad } from '../common/utils';
-import { test } from './fixture';
 import { expect } from '@playwright/test';
-import { chatTestSetup, sendMessage, waitForMessageSeen, waitForMessageWithContent } from '../common/chatTestHelpers';
+import {
+  chatTestSetup,
+  sendMessage,
+  waitForMessageSeen,
+  waitForMessageWithContent
+} from '../../common/chatTestHelpers';
+import { stubMessageTimestamps, waitForChatCompositeToLoad } from '../../common/utils';
+import { test } from '../fixture';
 
```

Comment:
The tests below: 
- on line 14 is just to load the composite with a live adapter to make sure our composite works with the real adapter
- on line 18 cannot be converted to fake adapters as refreshing the page to rejoin the chat would lose the state of the chat because the state is maintained in browser memory

Please let me know if either these tests should removed from our CI

Reply:
Lets remove in a seperate PR if we find them causing issues